### PR TITLE
Add comment about dependent packages

### PR DESCRIPTION
**Pull request by LMS007**
Remove browserify as a dependent package as its not actually required.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,8 +4,9 @@ A package of resources for Hypothesis front-end applications.
 
 #### Requirements
 
+Dependencies **required** for importing modules from `@hypothesis/frontend-shared`:
+
 - preact
```
Comment (LMS007) [inline README.md:9]: browserify is not really a requirement at all so I think it's best not to confuse this with a actual  dependency such as preact.

File: README.md [hunk 2]
```diff
@@ -4,8 +4,9 @@ A package of resources for Hypothesis front-end applications.
 
 #### Requirements
```
Comment (robertknight) [inline README.md:5]: The heading levels are a bit wonky here. We've got a level 1 heading and underneath that a level 4 sub-heading ("Requirements") followed by a level 3 heading ("Usage"). I would correct this so that there is a level 1 heading at the top with level 2 for the next logical level down and so on.

File: README.md (hunk 1 above)
Comment (robertknight) [reply README.md:9]: I would probably suggest rewording this section, including the list, as:

"Your project must have Preact installed as a dependency."

The main change here is just to make it clear that this is an unconditional dependency. npm/yarn will expect Preact to be installed whether or not you actually use any UI components, as long as preact is listed as a mandatory peer dependency. 

In writing the above comment I tried to find a link in the Preact docs that would tell you how to install Preact with yarn and npm but they miss that step, so perhaps we could add it here (`yarn install --save preact`).

**Review by robertknight — COMMENTED** [review]
I suggested a revised wording. Please let me know what you think.

File: README.md [hunk 3]
```diff
@@ -2,18 +2,19 @@
...
 
-#### Requirements
+## Requirements
 
-- preact
-- browserify
+Your project must have Preact installed as a dependency.
 
-### Usage
+`yarn install --save preact`
```
Comment (robertknight) [inline README.md:9]: Can I suggest changing this to `npm install` for consistency with the command below.

**Review by robertknight — APPROVED** [review]
LGTM with a change to use `npm` in both install commands for consistency, even though we use `yarn` for several of our projects.
